src/taxonomy/taxonomy-card/index.jsx

```diff
@@ -64,6 +44,47 @@ HeaderSubtitle.propTypes = {
   orgsCount: PropTypes.number.isRequired,
```

Comment:
I found a warning in the console as soon as the page loads:

```
Warning: Failed prop type: The prop `orgsCount` is marked as required in `HeaderSubtitle`, but its value is `undefined`.
    at HeaderSubtitle (webpack-internal:///./src/taxonomy/taxonomy-card/index.jsx:42:3)
    at TaxonomyCard (webpack-internal:///./src/taxonomy/taxonomy-card/index.jsx:119:3)
```
We probably should either remove the `isRequired` and set a `defaultProp`, or make sure that something is always passed in.

Reply:
Updated [here](https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/d2f5c65a054ec4b4aa295f8b107204a63d945184#diff-af21fd1bf60f49209dc1ae57b7314da8383efb95a70824cc1e00fdd8b5b4303eR41)